### DISCUSSION TITLE
Deleting a spam post should unsubscribe the victims.

### DISCUSSION THREAD

**Discussion by hwine** [community/community#42810] [Discussions]
### Select Topic Area

Product Feedback

### Body

My GitHub account was @ mentioned in a spam comment to a discussion in a repo with no connection to me. The repo maintainer quickly deleted the spam comment, however all of us who got mentioned were left subscribed to the discussion thread.

Since one of those folks had their out-of-office responder configured, that posted a discussion response, which initiated GitHub notification. This could easily turn into a flood 😞.

It would be great if, when a comment is deleted, any subscriptions due to mentions in the spam comment could be reversed.

I suspect the same would happen in a PR or Issue thread.

<details>
The discussion in question was Cascade-radios-games / Happy-Wolf-Viewer / discussions/5 # discussioncomment-4512861
</details>

**Comment by FurkanKalaycioglu** [comment]
I also received such email and notifications. I find it very weird that this type of things happens under GitHub as well.

**Comment by geckolinux** [comment]
I've received at least 10 different notifications during the past 24 hours from porn spammers on my Github project's issues as well. Github needs to implement some sort of rate restriction and privilege restriction on new Github user accounts, this can't continue.

Any official comments from a Github employee?

**Comment by Akash1134** [comment]
Hi everyone, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.
